src/System.Diagnostics.PerformanceCounter/src/System/Diagnostics/PerformanceCounterLib.cs

```diff
@@ -1237,7 +1244,6 @@ internal static void UnregisterCategory(string categoryName)
         {
             RegisterFiles(categoryName, true);
             DeleteRegistryEntry(categoryName);
-            CloseAllTables();
```

Comment:
Why is CloseAllTables no longer needed?

Reply:
CloseAllLibraries() call Close() on every PerformanceCounterLib instance that call CloseTables(). Isn't this  redundant?

Reply:
I don't know.  I'm not familiar with the codebase.

Reply:
I'm inclined to say that we should leave CloseAllTables in place.  I agree that the work is done implicitly by calling CloseAllLibraries, but some of this work is done outside of locks and I'd hate to change the characteristics of a race condition and break user code.  Given that this functionality is here for compatibility, I'd leave it.

Reply:
@brianrob ok